### PR TITLE
Add slide toggle for visualisation sets

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: src/components/mapgl/mapgl.component.css [hunk 1]
```diff
@@ -46,37 +46,84 @@
...
   flex-direction: column;
-  top: 250px;
-  right: 10px;
+  bottom: 165px;
+  right: 8px;
+  background: transparent;
+
+}
+.map_legend-button--wrapper {
+}
```
Comment (sebbousquet) [inline src/components/mapgl/mapgl.component.css:59]: empty class ?

File: src/components/mapgl/mapgl.component.css [hunk 2]
```diff
@@ -46,37 +46,84 @@
...
+.map__visu-list--container /* @doc Sets the style of layers switcher container*/
+ {
   background: white;
-  padding: 5px;
+  padding: 1px;
   box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2), 0 2px 2px 0 rgba(0, 0, 0, 0.14),
     0 1px 5px 0 rgba(0, 0, 0, 0.12);
   border-radius: 2px;
+  max-height: 590px;
```
Comment (sebbousquet) [inline src/components/mapgl/mapgl.component.css:67]: This height is relevant on lot of resolutions ?

Comment (MohamedHamouGisaia) [reply src/components/mapgl/mapgl.component.css:67]: `bien vu`, it's not, I'll fix it

File: src/components/mapgl/mapgl.component.css (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/components/mapgl/mapgl.component.css]: fixed

File: src/components/mapgl/mapgl.component.css (hunk 2 above)
Comment (MohamedHamouGisaia) [reply src/components/mapgl/mapgl.component.css]: fixed

**Pull request merged**